assets/js/vendor/cards-expandable.js

```diff
@@ -0,0 +1,12 @@
+// JS for expandable cards (.va-card--expands)
+$(document).ready(function() {
+	$('.va-card-body').on('click', function(e){
+    	if(e.target.nodeName == 'DIV') {
```

Comment:
This is a bit of a weird mix between jQuery and regular JavaScript. I see why you wanted to use `e.target` instead of `this` because you want to check if what was clicked on was the whole card or just the header, but it reads a little weird. I see why you would want to use jQuery, because it does make some of the handling of checking for and changing classes easier. I wonder if it's better to use `$(e.target).is('div')` instead? I was also thinking that you could use `this` when checking the card itself, but maybe best to make your 2 checks here consistent. Not sure what the best solution is, just throwing out ideas.